cookbook/providers/sambanova/basic_stream.py

```diff
@@ -0,0 +1,14 @@
+import os
+from phi.agent import Agent, RunResponse  # noqa
+from phi.model.sambanova import Sambanova
+
+api_key = os.getenv("SAMBANOVA_API_KEY")
```

Comment:
Quick question: If `Sambanova` class includes `api_key: Optional[str] = getenv("SAMBANOVA_API_KEY")`, do we need this line in the cookbooks?